pkg/neorpc/result/version.go

```diff
@@ -15,12 +15,13 @@ type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
-		TCPPort   uint16   `json:"tcpport"`
-		WSPort    uint16   `json:"wsport,omitempty"`
-		Nonce     uint32   `json:"nonce"`
-		UserAgent string   `json:"useragent"`
-		Protocol  Protocol `json:"protocol"`
-		RPC       RPC      `json:"rpc"`
+		TCPPort     uint16      `json:"tcpport"`
+		WSPort      uint16      `json:"wsport,omitempty"`
+		Nonce       uint32      `json:"nonce"`
+		UserAgent   string      `json:"useragent"`
+		Protocol    Protocol    `json:"protocol"`
+		Application Application `json:"application"`
```

Comment:
@roman-khimov need your ACK for this. I suggested moving it to a separate section because both SaveInvocations and KeepOnlyLatestState belong to application configuration (not protocol or RPC). An alternative to `application` is `ledger`.

The problem is that it's a completely new section comparing to C# response, so I'm not sure we're allowed to do this. There will be no parsing problems with C# RPC client.

Reply:
It's a nice counterpart for `protocol`, follows configuration as well:
https://github.com/nspcc-dev/neo-go/blob/2971b8a1c7dfc8cb895f9a60e8f1f36c60020948/pkg/config/config.go#L51-L56

ACK